data-prepper-plugins/http-source/README.md

```diff
@@ -33,6 +41,7 @@ source:
 - `successRequests`: measures total number of requests successfully processed by HTTP source plugin.
 - `badRequests`: measures total number of requests with invalid content type or format processed by HTTP source plugin.
 - `requestTimeouts`: measures total number of requests that time out in the HTTP source server.
+- `requestsTooLarge`: measures total number of requests of which the events size in the content is larger than the buffer capacity.
```

Comment:
It might be nice to directly mention the HTTP response status numbers that correspond to each of these metrics